client/lib/signup/step-actions.js

```diff
@@ -222,7 +239,9 @@ function getUsernameSuggestion( username, reduxState ) {
 }
 
 module.exports = {
-	createSiteWithCart: createSiteWithCart,
+	createCart,
+
+	createSiteWithCart,
```

Comment:
If we use the shorthand syntax here then we should probably update the rest of exports as well.